yt/visualization/volume_rendering/scene.py

```diff
@@ -367,6 +367,7 @@ def save_annotated(
         dpi: int = 100,
         sigma_clip: Optional[float] = None,
         render: bool = True,
+        tf_rect: Optional[list] = None,
```

Comment:
requires `from typing import List` at the top level
```suggestion
        tf_rect: Optional[List[float]] = None,
```